eos/b-decays/b-to-dstar-l-nu.cc

```diff
@@ -77,11 +241,36 @@ namespace eos
 
             u.uses(*form_factors);
             u.uses(*model);
+
+        }
+
+        // normalization according to DSD2014
```

Comment:
maybe add also equation number when possible:
according to DSD2014 eq.(7)

Reply:
Yes! Please use the format
´´´
cf. [REFERENCE], eq. (XX), p. YY
´´´
or
´´´
cf. [REFERENCE], eqs. (XX), p. YY
´´´

Reply:
okay